sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RestrictDeserializedTypes.cs

```diff
@@ -37,7 +37,7 @@ internal void BinderAsVariable(Stream stream, bool condition)
 
             var formatter2 = new BinaryFormatter();
             formatter2.Binder = unsafeBinder;
-            formatter2.Deserialize(stream); // Noncompliant: unsafe binder used
+            formatter2.Deserialize(stream); // Noncompliant [unsafeBinder1]: unsafe binder used
```

Comment:
what is `unsafeBinder1` used for?

Reply:
This is a new syntax improvement added by @pavel-mikula-sonarsource and @christophe-zurn-sonarsource which allows to link a line of code to a specific secondary location.